jmclient/jmclient/configure.py

```diff
@@ -217,6 +217,22 @@ def jm_single():
 # 1,000,000 satoshis.
 absurd_fee_per_kb = 350000
 
+# In decimal, the maximum allowable change either lower or
+# higher, that the fee rate used for coinjoin sweeps is
+# allowed to be.
```

Comment:
Nit: I'm curious if changing 
"that the fee rate used for coinjoin sweeps is allowed to be"
to
"that the fee rate used for coinjoin sweeps is allowed to **have**"
is correct, equal or wrong.

I read it as in "The fee rate for sweeps is allowed to have a maximum change of {max_sweep_fee_change}"

Reply:
"Be" should be correct, I think "have" would also be fine. That it is complex/ambiguous I understand, and that's why I gave a concrete example there.